packages/zosfiles/src/api/constants/ZosFiles.messages.ts

```diff
@@ -443,6 +443,14 @@ export const ZosFilesMessages: { [key: string]: IMessageDefinition } = {
         message: "Invalid zos-files create command 'perms' option: "
     },
 
+    /**
+     * Message indicating that the USS file or directory was created successfully
+     * @type {IMessageDefinition}
+     */
+    ussFileDirectorySuccessfully: {
```

Comment:
This message has the same message as ussCreatedSuccessfully. Please choose one and remove duplication, or alternatively consider creating two separate messages for USS file created successfully and USS directory created successfully.